__tests__/app.test.js

```diff
@@ -35,7 +36,51 @@ afterAll(() => db.end());
             .get('/api')
             .expect(200)
             .then(({body}) => {
-                expect(body).toEqual(endPoints)    
+                expect(body.endpoints).toEqual(endPoints)    
+            })
+        })
+    })
+
+    describe('/api/articles', () => {
+        it('should get an article by _Id and respond with the corresponding article object with the correct properties', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({body}) => {
+                const article = body.article
+                console.log(article)
+                expect(article.length).toEqual(1)
+                article.forEach((element) => {
```

Comment:
you don't need to iterate through individual articles if you know there's only one